alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultAuditUtility.java

```diff
@@ -346,10 +345,10 @@ private NotificationConfig populateConfigFromEntity(AlertNotificationModel notif
         return new NotificationConfig(id, createdAt, notificationEntity.getProvider(), providerConfigId, providerConfigName, providerCreationTime, notificationEntity.getNotificationType(), notificationEntity.getContent());
     }
 
-    private Date parseDateString(String dateString) {
-        Date date = null;
+    private OffsetDateTime parseDateString(String dateString) {
+        OffsetDateTime date = null;
         try {
-            date = DateUtils.parseDate(dateString, DateUtils.AUDIT_DATE_FORMAT);
+            date = DateUtils.parseDate(dateString, DateUtils.UTC_DATE_FORMAT_TO_MINUTE);
```

Comment:
In a separate conversation you mentioned that DateUtils.AUDIT_DATE_FORMAT was used in a test only, but this class was using the format. The method that uses the parseDateString is converting to paged models.  I believe it is being used. Is this correct to replace this format with the less precise minute format?

Reply:
Yeah, I must have missed this reference in the refactor. I think you are correct and we should keep it.